index.html

```diff
@@ -0,0 +1,214 @@
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/lecture-talk.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Lecture</h2>
+              <p>Attend the lectures of the web development course training.</p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/class-project.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Projects</h2>
+              <p>
+                Finish class projects to train yourself what you learned in the
+                lecture of the course.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/workshop.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Workshop</h2>
+              <p>
+                Practice web development techniques in our workshop session.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/lecture.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Forum</h2>
+              <p>
+                Have the time to share your opinions and your thoughts with
+                expert developers.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/certification.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Certification</h2>
+              <p>
+                Get a web development certificate verfied by St. Mary's
+                University.
+              </p>
+            </div>
+          </div>
+          <button type="button" class="register-btn">
+            Register For The Training
+          </button>
+          <a href="#" class="see-all-program">See the Whole Program</a>
+        </div>
+      </section>
+      <section id="trainers">
+        <div class="desktop-padding">
+          <h2 class="trainers-section-title the-underline">Trainers</h2>
+          <div id="trainers-container"></div>
+          <button id="more-trainers" class="more-trainers">
+            More <img src="images/icons/down.svg" alt="See more icon" />
+          </button>
+        </div>
+      </section>
+      <section id="partners">
+        <h2 class="partners-section-title the-underline">Partner</h2>
+        <div class="partners-container">
+          <img
+            class="partner-image"
+            src="images/social/airbnb.svg"
+            alt="Airbnb"
+          />
+          <img
+            class="partner-image"
+            src="images/social/facebook.svg"
+            alt="Facebook"
+          />

```

Comment:
- [ ] Kindly use a `ul` with each image in an `li` tag as this is the right semantic tag

Reply:
- I think you should wrap your images in list tag` (<ul> and <li>)`. Can we implement this?? 🤗